docker/Dockerfile

```diff
@@ -47,8 +48,8 @@ WORKDIR /usr/src/app
 ######################
 # DEPENDENCIES START #
 ######################
-# Env vars needed for dependency install and asset precompilation
-ENV RAILS_ENV='production'
+# Env vars needed for dependency install and asset precompilation?? maybe not
```

Comment:
Do we know this now, should we remove the comment if so?

Reply:
Not sure so just added the comment as this was yak shaving issues coming up. But from my understanding a single docker command like this does not make the variable available to subsequent commands (i.e the RUN bundle install command in particular). We are handling production-only gem install via the bundle command directly now so probably not needed.

you could have a play and chck whether removing this line on a separate branch and then build changes the gems installed. I am pretty sure it won't.